src/applications/personalization/rated-disabilities/actions/index.js

```diff
@@ -45,15 +45,29 @@ export function fetchRatedDisabilities() {
   };
 }
 
+function getResponseError(response) {
```

Comment:
there have been times I've seen an API error come back in this form:
```
{
  "status": 500,
  "error": "Internal Server Error",
  "exception": {}
}
```
So we need to be able to handle a situation like that; ie, an error that _doesn't_ include a nicely package `errors` array.